src/main/java/com/dumbdogdiner/stickyapi/common/webapis/HttpException.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.common.webapis;
+
+import java.net.URL;
+
+public class HttpException extends RuntimeException {
```

Comment:
Believe I've commented on this before - why the need for two layers of wrapping?

Reply:
It's one layer of wrapping in most cases, but its using inheritance to subclass different types of exception so you can do different behaviors